cli/client.go

```diff
@@ -490,7 +490,7 @@ func newViamClient(c *cli.Context) (*viamClient, error) {
 	}
 
 	if conf.BaseURL != defaultBaseURL {
-		infof(c.App.Writer, "Using %q as base URL value", conf.BaseURL)
+		infof(c.App.ErrWriter, "Using %q as base URL value", conf.BaseURL)
```

Comment:
Justification: This is the worst offender because it sits in front of every cli subcommand so if we want script against our cli, we have to be resilient to this message and remove it from the stdout if it exists (or just fail for non-default base-urls).

I think this clearly falls into the category of "diagnostic info"

Reply:
> Additionally, this really feels like this should only be printing at all when some sort of "debug" is enabled.

This msg has certainly saved many hours of debugging time within viam because people don't use the cli enough to remember if they are logged into staging or prod and before this line was added, would post very confused messages in slack about not being able to find their org or missing robots, etc. I think it should continue to not be hidden behind a flag, I just dont think that it belongs on stdout.

Reply:
Sounds like a fair opinion wrt debug vs. info. Fine with `info` and `ErrWriter`.